CHANGES.rst

```diff
@@ -35,6 +35,9 @@ astropy.io.misc
 ^^^^^^^^^^^^^^^
 - Added serialization of parameter constraints fixed and bounds.  [#10082]
 
+- Fix ASDF serialization of circular model inverses, and remove explicit calls
+  to ``asdf.yamlutil`` functions that because unnecssary in asdf 2.6.0. [#10189]
```

Comment:
`because they are unnecessary` ?

Reply:
Oops, thanks, I intended to write "that became unnecessary" but went off the rails.